### PR TITLE
Vision

### PR DESCRIPTION
**Pull request by swhitla**
For discussion

### CODE REVIEW

File: product_vision_and_principles.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## What is this?
+
+In its first few years, the Shared Meaning Platform (SMP) has struggled to evolve a clear set of design principles and philosophy, given the number of different directions and uses it has been put to on different client projects. This document sets out some Guiding Principles that are not going to change, in order to:
+
+1. accelerate the development of design principles by setting some business-level constraints
+2. help the team have more meaningful conversations about the decisions we need to make in the meantime
+
+## Some context
+
+VM is not a software business. We could become one - we have enough use cases to go down the traditional VC route, attract investment, build a larger team, grow a user base, develop the tool and sell the company.
```
Comment (rosi-grafixoft) [inline product_vision_and_principles.md:12]: Not sure that "sell the company" should be here. Not all software companies are build to be sold. Still modern business ideas are  not possible without "platform/software infrastructure", Unless you want to print the maps and sell it as a board game (hm, this is cool new idea). If you want to create meaning and present it to larger group of people you need  a way to do it. When I worked in the bank we had around 400 people in the branches and more than 300 people in the IT departments. You need a "tribune" to say what you need to be said unless you want to make а  boutique meaning. :)

File: product_vision_and_principles.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+But this isn't our purpose. We were established to try to answer a much bigger question, which is how our clients, and ultimately the world at large, can create enough shared meaning to have a chance of solving the really big systemic challenges that humans struggle to articulate, let alone tackle. This is a long term goal, and we are only at the beginning. Our approach, in other words, is the opposite of the usual short-termism associated with VC or PE funded start-ups; we are only willing to grow at the rate that we find clients who are aligned with this purpose, and willing to partner with us to seek answers together.
```
Comment (rosi-grafixoft) [inline product_vision_and_principles.md:14]: so the clients will be investors?

File: product_vision_and_principles.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+- Core platform features:
+	- should be at least as simple and intuitive to use for non-experts as Google Maps
+	- should be optimised, stable, and not change very much.
+	- have two primary use cases:
+		1. to help new folks understand how their organisational system/eco-system works and where they fit into it
+		2. to help everyday folks find organisational (as opposed to operational) information that would otherwise be hard to track down because it lives on a bunch of different corporate systems (e.g. Who owns this process? When does the contract for this IT system expire? What does this acronym stand for? Who else does this activity? etc.)
+	- should cover the following interactions and nothing else: search, map switching, map exploration, item selection, link navigation
+	- should also include an "advanced" mode for expert users who want to perform more advanced filters / queries directly on the model. This should include list mode, graph mode (?), filter tools, advanced search, detail level control. How this integrates with the basic experience is an ongoing design challenge; whatever choices we make must not jeopardise the simplicity of the experience for new users.
```
Comment (rosi-grafixoft) [inline product_vision_and_principles.md:43]: I don't like the "advance mode". If this is only for internal use, ok. But google maps doesn't have such mode. I prefer to have good UX design and help page with some nice videos maybe instead to hide the functionalities under some modes.

File: product_vision_and_principles.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+	3. We help **change agents** take advantage of this shared meaning by designing custom interactions on the SMP to support systems interventions (e.g. telling stories, asking questions, visualising patterns ...)
+	4. As stakeholders become more engaged with the platform, they become "change agents" themselves, using more of the change agent tools to support interventions of their own, thus creating a virtuous cycle in which more shared meaning is generated across larger ecoysystems, unlocking the potential for more significant system changes
+- We will continue to run a smaller proportion of projects outside of these long term partnerships, but they will still be based on the same theory of change, i.e. generating shared meaning from consistent base maps, on which interactions can be designed that are tailored to the intervention need 
+
+The principle is that this theory of change needs to be reflected in our design decisions, particularly the distinction between "core platform features" for stakeholders and "customisable tools" for change agents, as described in the next principle.
+
+## Guiding Principle 2: Separate core from extensible functionality
```
Comment (rosi-grafixoft) [inline product_vision_and_principles.md:34]: I some how did not get which is core and which is extensible

File: product_vision_and_principles.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+- Customisable tools:
+	- should be based on a set of simple unified design principles / system, applied to an extensible set of discrete, configurable interaction "behaviours"
+	- have two primary use cases:
+		1. to allowing change agents to build "tasks" for users to perform in support of change intiatives
+		2. to configure custom states for specific user groups (e.g. a view with particular pre-set filters to be used as a decision support tool) This includes default states that could be applied to larger audiences e.g. having certain filter toggles always on
+
+## Guiding Principle 3: Promote autonomy down the user chain
+
+- The business model is based on the assumption that the use of the tool will develop organically over time, so we achieve our purpose best when we make it easy/desirable for people to move from one "level" of user to the next.
+- We can think of users in a "chain": from devs, to VM consultants, to "change agents", to "stakeholders". When we develop features, we should develop them in a way that maximises autonomy for the next user on in the chain.
```
Comment (rosi-grafixoft) [inline product_vision_and_principles.md:53]: I think that my vision about the user is different. User is the person who is not part of building the map or the platform. The user is only who consume the meaning. And if the end user can't "consume" then the stakeholder/client won't be interested in investing in such tool. Can you please add the end user here. :)

**Review by bz2 — APPROVED** [review]
I think this is worth landing as a good articulation of some of the foundational ideas behind what VM is aiming to make.

Some of my questions are similar to those Rosi has raised here around differences of opinion about what a good product ends up looking and feeling like, but these elements ideally can live somewhere else that's more malleable.

**Comment by swhitla** [comment]
The thinking underlying the text has moved on a good bit in the intervening period, but let's land it for now and update later in the year.

**Pull request merged**